extensions/gitlab-docs/src/index.tsx

```diff
@@ -0,0 +1,144 @@
+import { ActionPanel, Action, List, showToast, Toast } from "@raycast/api";
+import { useState, useEffect, useRef, useCallback } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+const apiUrl = "https://search-api.swiftype.com/api/v1/public/installs/DTF81Pizm7yGQpgXcrMP/suggest.json";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List
+      isLoading={state.isLoading}
+      onSearchTextChange={search}
+      searchBarPlaceholder="Search GitLab Handbook..."
+      throttle
+    >
+      <List.Section title="Results" subtitle={state.records.length + ""}>
+        {state.records.map((searchResult, index) => (
+          <SearchListItem key={index} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      icon="../assets/handbook-icon.png"
+      title={searchResult.name}
+      accessoryTitle={searchResult.category}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.OpenInBrowser title="Show Details" url={searchResult.url} />
```

Comment:
This title is usually used when opening the Detail Panel; maybe use the default title?